### DISCUSSION TITLE
Detailed documentation needed

### DISCUSSION THREAD

**Discussion by p6002** [mudler/LocalAI#429] [Q&A]
It is written about OpenAI, later to install some models. What are the models that don't require a GPU to work properly? However, on screenshots I see GPT-3.5
I got lost.

```
# copy your models to models/
cp your-model.bin models/
```

**Comment by sebastien-prudhomme** [comment]
@p6002 The easiest way to install models is to use the gallery, see examples: https://github.com/go-skynet/model-gallery.

**Comment by Aisuko** [comment]
Thanks for your feedback. As @p6002 mentioned, gallery project is useful. And here is our new document website. Here is the quick start part. https://localai.io/basics/getting_started/index.html
